Create rowlock.md (#41)
Virtual-Viking-Longship/Ship-Annotations: annotations/rowlock.md

**_rowlock_** (English); _åretol_ (Danish); _Riemendolle_ (German)

_**keipr** m., pl. keipar_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o42936)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=44158)]  

> Curved wood on the upper edge of boats’ sides, which allowed for rowing and for oars to be rested. A tholepin was used as a fulcrum for oars on smaller vessels. 

Reviewer: ```suggestion
A fulcrum for oars on smaller vessels, where oarholes would not be practical. 
```

<div align="center">
  
  ![rowlocks from Gokstad ship](../images/Rowlocks_Gokstad.png)
  

>   rocklocks from the Gokstad ship (Nicolaysen Pl. V, Fig. 5,6,9,13,15,)

Reviewer: ```suggestion
  Carved rowlocks from the boats in the Gokstad ship, with holes drilled to accept the oar-grommets (Nicolaysen Pl. V, Fig. 5,6,9,13,15)
```

> The various methods of mounting rowlocks were probably primarily determined by the thickness of the ships’ sheerstrakes (Crumlin-Pedersen, 126). 

Reviewer: ```suggestion
In the Viking Age, the oars of small boats were usually pulled against curved wood on the upper strake of boats’ sides, and/or thole pins driven into holes in the gunwale. Both were often held in place by oar-grommets (ON _ hömlu-bönd_). The various methods of mounting rowlocks were probably primarily determined by the thickness of the ships’ sheerstrakes (Crumlin-Pedersen, 126). 
```

---

> Jesch, Judith. _Ships and Men in the Late Viking Age: The Vocabulary of Runic Inscriptions and Skaldic Verse._ NED-New edition. Woodbridge, Suffolk, UK ; Rochester, NY: 

Reviewer: ```suggestion
Nicolaysen, N. _Langskibet fra Gokstad ved Sandefjord_. Kristiania: Alb. Cammermeyer, 1882.
```
